etl_manager/meta.py

```diff
@@ -852,3 +852,62 @@ def get_existing_database_from_glue_catalogue(database_name):
         name=database_name, bucket=bucket, description=database_description
     )
     return db
+
+
+def _parquet_metadata_type_to_etl_mgr_type(pmeta_type):
+    """
+    Convert a field from parquet metadata dictionary to a etl_manager type string 
+    """
+
+    if type(pmeta_type) == str:
+        type_string = pmeta_type
+
+    # If it's not a string, it's a complex type
+    elif pmeta_type["type"] == "struct":
+        fields = pmeta_type["fields"]
+        type_items = []
+        for field in fields:
+            key = field["name"]
+            # each field can itself be a struct or array, so recurse
+            etl_type = _parquet_metadata_type_to_etl_mgr_type(field["type"])
+            type_items.append(f"{key}:{etl_type}")
+        type_items_string = ",".join(type_items)
+        type_string = f"struct<{type_items_string}>"
+
+    elif pmeta_type["type"] == "array":
+        element_type = pmeta_type["elementType"]
+        # elements of an array can themselves be a struct or array, so recurse
+        et_string = _parquet_metadata_type_to_etl_mgr_type(element_type)
+        type_string = f"array<{et_string}>"
+
+    # Convert etl_manager agnostic types to glue/athena datatypes
+    for key, value in _agnostic_to_glue_spark_dict.items():
+        type_string = type_string.replace(value["glue"], key)
+    type_string = type_string.replace("integer", "int")
```

Comment:
I think this step should be applied to `etl_type`. before it is appended to the `type_items` list (i.e. line 872 - don't think this applies to arrays (line 880) but worth checking). Atm if you have a column name that matches the agnostic type you would change the column name in the meta? 

e.g. if I had an struct column with a key named `character` then the meta should be `struct<character:string>` but this would return `struct<string:string>`?

(Would probably be good to add a test for the above as well)